app/components/cards/featured_story_component.rb

```diff
@@ -0,0 +1,57 @@
+module Cards
+  class FeaturedStoryComponent < CardComponent
+    def initialize(card:, page_data:)
+      super(card: card)
+      @page_data = page_data
+    end
+
+    def link
+      story.path
+    end
+
+    def link_text
+      @link_text || "Read #{story_name}'s story"
```

Comment:
Better to use `pluralize` here? in case you have `Read Ross's story`

Reply:
Also I'm a bit confused at what this is doing, won't `@link_text` always be `nil`?

Reply:
No, with inheritance - it could be overridden in the card definition, that being said this should be `@link_text ||=`